packages/x-components/src/__stubs__/popular-searches-stubs.factory.ts

```diff
@@ -0,0 +1,34 @@
+import { Suggestion } from '@empathyco/x-types';
+
+/**
+ * Creates a list of generic popular searches.
+ *
+ * @param amount - The number of popular searches to create.
+ *
+ * @returns An array of popular searches.
+ */
+export function getPopularSearchesStub(amount = 3): Suggestion[] {
+  return Array.from({ length: amount }, (_, index) =>
```

Comment:
`Array.from<number, Suggestion>`

Reply:
Not needed. The fake array you are creating doesn't need any type, which in your case is the `number` type. And as you are returning if you don't fulfill the `Suggestion[]` type TS will complain. You earn nothing but writing more code here.

Reply:
I was trying to follow the same approach in all the stubs, we have most of then with the `<type>`